addons/mail/models/discuss/mail_guest.py

```diff
@@ -124,7 +126,7 @@ def _field_store_repr(self, field_name):
         return [field_name]
 
     def _to_store_defaults(self):
-        return ["avatar_128", "im_status", "name"]
+        return ["avatar_128", "im_status", "last_poll", "name"]
```

Comment:
Have `_field_store_repr` of `im_status` return both fields, don't manually add `last_poll` everywhere.

Reply:
This also must be done for partner.

Reply:
Yes this is the idea, first array item can just be "im_status" as it is an actual field (it shouldn't loop).